README.md

```diff
@@ -9,23 +9,40 @@ cycle.
 
 # Install and run the website locally (for development)
 
-_NB: The following instructions are for Linux.
-On Mac, Ruby version 2.6 may be necessary. You can install a specific version
-with [rbenv](https://github.com/rbenv/rbenv)._
-
 1. Clone the repository, move to its directory
     ```
     git clone git@github.com:eWaterCycle/ewatercycle.github.io.git
     cd ewatercycle.github.io
     ```
+1. Run with docker (recommended) or install Ruby and run locally
+
+## Install and run with Docker
+
+1. Ensure Docker is running.
```

Comment:
I'm not sure I understand what this means.

Reply:
On my OS, I needed to first turn on Docker before running a docker command from the command line. Does this work differently on Linux?
(And more to the point: what would be a better way to phrase this?)

Reply:
So how do you turn it on, or confirm that it's running?